cid-redirects.json

```
@@ -4051,5 +4051,12 @@
   "/docs/integrations/amazon-aws/aurora-mysql-ulm": "/docs/integrations/amazon-aws/rds",
   "/docs/integrations/amazon-aws/aurora-postgresql-ulm": "/docs/integrations/amazon-aws/rds",
   "/docs/integrations/amazon-aws/elastic-load-balancer-app": "/docs/integrations/amazon-aws/application-load-balancer",
-  "/docs/integrations/amazon-aws/elastic-load-balancing-classic": "/docs/integrations/amazon-aws/classic-load-balancer"
+  "/docs/integrations/amazon-aws/elastic-load-balancing-classic": "/docs/integrations/amazon-aws/classic-load-balancer",
+  "/docs/manage/scheduled-views": "/docs/manage/views",
+  "/docs/manage/scheduled-views/add-scheduled-view": "/docs/manage/views/add-scheduled-view",
+  "/docs/manage/scheduled-views/pause-disable-scheduled-views": "/docs/manage/views/pause-disable-scheduled-views",
+  "/docs/manage/scheduled-views/run-search-against-scheduled-view": "/docs/manage/views/run-search-against-scheduled-view",
+  "/docs/manage/scheduled-views/scheduled-view-lag-time": "/docs/manage/views/scheduled-view-lag-time",
+  "/docs/manage/scheduled-views/scheduled-views-best-practices": "/docs/manage/views/scheduled-views-best-practices",
+  "/docs/manage/scheduled-views/view-list-scheduled-views": "/docs/manage/views/view-list-scheduled-views"
```

Do these need to be adjusted? 

```suggestion
  "/docs/manage/scheduled-views/add-scheduled-view": "/docs/manage/views/add-view",
  "/docs/manage/scheduled-views/pause-disable-scheduled-views": "/docs/manage/views/pause-disable-view",
  "/docs/manage/scheduled-views/run-search-against-scheduled-view": "/docs/manage/views/run-search-against-view",
  "/docs/manage/scheduled-views/scheduled-view-lag-time": "/docs/manage/views/view-lag-time",
  "/docs/manage/scheduled-views/scheduled-views-best-practices": "/docs/manage/views/best-practices",
  "/docs/manage/scheduled-views/view-list-scheduled-views": "/docs/manage/views/view-list"
```

---

Yes, they need to be fixed. The build report says:
>  These paths are redirected to but do not exist:
>  - /docs/manage/scheduled-views/
>  - /docs/manage/scheduled-views/scheduled-views-best-practices/
>  - /docs/manage/scheduled-views/add-scheduled-view/
>  - /docs/manage/scheduled-views/run-search-against-scheduled-view/